pkg/cmd/container/list.go

```diff
@@ -53,14 +54,14 @@ func List(ctx context.Context, client *containerd.Client, options types.Containe
 //   - all means showing all containers (default shows just running).
 //   - lastN means only showing n last created containers (includes all states). Non-positive values are ignored.
 //     In other words, if lastN is positive, all will be set to true.
-func filterContainers(ctx context.Context, client *containerd.Client, filters []string, lastN int, all bool) ([]containerd.Container, error) {
+func filterContainers(ctx context.Context, client *containerd.Client, filters []string, lastN int, all bool) ([]containerd.Container, map[string]string, error) {
```

Comment:
can you add a comment explaining that, `formatter.ContainerStatus(ctx, c)` is time consuming so we do it in goroutines and return the result as a map.

Also can you update the PR description with the latest benchmark result from this implementation.

Reply:
done, thank you.